Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:4.1
+// swift-tools-version:4.0
```

Comment:
does it still work when built with swift 4.1?...

Reply:
This will still work. The swift-tools-version specifies the minimum version of Swift required for the project but higher versions are allowed.